lib/SIL/Utils/MemAccessUtils.cpp

```diff
@@ -419,6 +345,125 @@ bool swift::memInstMustInitialize(Operand *memOper) {
   }
 }
 
+bool swift::isSingleInitAllocStack(AllocStackInst *asi,
+                                   SmallVectorImpl<Operand *> &destroyingUses) {
+  // For now, we just look through projections and rely on memInstMustInitialize
```

Comment:
Are all the changes in this file, just moving things around? Without substantial changes?

Reply:
Purely code shuffling to make it possible to stage in other changes.